### DISCUSSION TITLE
More commands for ekg-notes-mode-map

### DISCUSSION THREAD

**Discussion by jayrajput** [ahyatt/ekg#30] [Ideas]
I wonder if more commands can be added to `ekg-notes-mode-map` and if they should be added by default to `ekg.el`. This discussion node is to capture any such suggestions.

I would like to see `kill-current-buffer` bind to `q`. I have added to my setup, but I wonder if adding to the default `ekg.el` increases the value proposition for a normal user.

**Comment by ahyatt** [comment]
This seems pretty reasonable to me, thanks for the suggestion.
